src/components/Swagger/Link.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import classes from './Swagger.module.scss'
+
+class Link extends React.Component {
+  handleClick = () => {
+    this.props.onClick(this.props.to.split('/')[2])
```

Comment:
This seems a bit hacky. Why the second route?

Reply:
The value which we receive is in the format 
```#/definitions/POI``` 
```#/definitions/POIRequest```

Reply:
I suppose the passed value should be formatted before to be used.